plugin/evm/gossiper_orders.go

```diff
@@ -85,18 +86,24 @@ func (n *pushGossiper) gossipSignedOrders() (int, error) {
 		return 0, nil
 	}
 
-	return len(selectedOrders), n.sendSignedOrders(selectedOrders)
+	err := n.sendSignedOrders(selectedOrders)
+	if err != nil {
+		n.stats.IncSignedOrdersGossipSendError()
+	}
+	return len(selectedOrders), err
 }
 
 func (n *pushGossiper) sendSignedOrders(orders []*hu.SignedOrder) error {
 	if len(orders) == 0 {
 		return nil
 	}
 
-	ordersBytes, err := rlp.EncodeToBytes(orders)
+	var buf bytes.Buffer
+	err := gob.NewEncoder(&buf).Encode(&orders)
```

Comment:
So not using rlp encoding anymore correct? What is the name of this encoding?

Reply:
this is gob encoding, part of the standard go library. RLP was part of go-ethereum